### PR TITLE
Provide `Microsoft.AspNetCore.Mvc.Testing` package README

### PR DESCRIPTION
**Pull request by MackinnonBuck**
Contributes to #48392

### CODE REVIEW

File: src/Mvc/Mvc.Testing/src/PACKAGE.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
+## About
+
+`Microsoft.AspNetCore.Mvc.Testing` provides support for writing integration tests for MVC applications.
```
Comment (martincostello) [inline src/Mvc/Mvc.Testing/src/PACKAGE.md:3]: Maybe remove the MVC specifier? It's also useful for Minimal APIs.

File: src/Mvc/Mvc.Testing/src/PACKAGE.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+### Configuration
+
+To configure the test app, follow these steps:
+
+1. Specify the Web SDK in the test project file (`<Project Sdk="Microsoft.NET.Sdk.Web">`).
+2. Add references to the following packages:
+    * `xunit`
+    * `xunit.runner.visualstudio`
+    * `Microsoft.NET.Test.Sdk`
+3. Expose the internal types from the web app to the test project. This can be done in the SUT's project file (`.csproj`):
```
Comment (martincostello) [inline src/Mvc/Mvc.Testing/src/PACKAGE.md:32]: This isn't a necessary step.

File: src/Mvc/Mvc.Testing/src/PACKAGE.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+                response.Content.Headers.ContentType.ToString());
+        }
+    }
+    ```
+
+## Additional Documentation
+
+For additional documentation and examples, refer to the [official documentation](https://learn.microsoft.com/aspnet/core/test/integration-tests) on integration testing in ASP.NET Core.
+
+## Feedback & Contributing
```
Comment (martincostello) [inline src/Mvc/Mvc.Testing/src/PACKAGE.md:76]:
```suggestion
## Feedback &amp; Contributing
```

File: src/Mvc/Mvc.Testing/src/PACKAGE.md (hunk 1 above)
Comment (javiercn) [reply src/Mvc/Mvc.Testing/src/PACKAGE.md:3]: Yes, but it was created for MVC, and it lives on the MVC namespace.

**Review by javiercn — APPROVED** [review]

File: src/Mvc/Mvc.Testing/src/PACKAGE.md (hunk 1 above)
Comment (MackinnonBuck) [reply src/Mvc/Mvc.Testing/src/PACKAGE.md]: I've updated the wording to:

> `Microsoft.AspNetCore.Mvc.Testing` provides support for writing integration tests for ASP.NET Core apps using MVC or Minimal APIs.

**Pull request merged**